src/Hint/Naming.hs

```diff
@@ -104,6 +104,6 @@ suggestName x
 
 
 replaceNames :: Data a => [(String,String)] -> a -> a
-replaceNames rep = descendBi f
+replaceNames rep = transformBi f
```

Comment:
I am unconvinced this changes anything. Before it looked at all the Ident's within the data type. Now it also looks at all the idents inside indents. Does it have any change for you?

Reply:
Ah, forgot to rewind this in the second commit, which changes the refactoring from Decl to Bind. Will do so in the new commit.